src/embed/types.rs

```diff
@@ -90,6 +92,87 @@ pub struct GetResponse {
     pub has: bool, // Second to avoid trailing comma if value == None.
 }
 
+#[derive(DeJson)]
```

Comment:
I think i would have expected to see some dispatch tests for this in https://github.com/rocicorp/repc/blob/master/tests/wasm.rs. Unless there is a technical reason why we can't have them seems like in addition to the scan unit tests in db we probably want some coverage at the higher level as well.

Reply:
You got it.